patches/tModLoader/Terraria.ModLoader.Core/LogArchiver.cs

```diff
@@ -40,8 +54,8 @@ internal static class LogArchiver
 
 		private const int MAX_LOGS = 20;
 		private static void DeleteOldArchives() {
-			var pattern = new Regex(".*\\.zip");
-			var existingLogs = Directory.GetFiles(Logging.LogDir).Where(s => pattern.IsMatch(Path.GetFileName(s))).OrderBy(File.GetCreationTime).ToList();
+			var pattern = new Regex(".*\\.zip");
+			var existingLogs = Directory.GetFiles(Logging.LogArchiveDir).Where(s => pattern.IsMatch(Path.GetFileName(s))).OrderBy(File.GetCreationTime).ToList();
```

Comment:
Regex can be replaced with wildcards here too, not sure why it was in the original file.